src/utils/adapters.js

```diff
@@ -400,8 +401,10 @@ export function adaptTokenEvents(events, blockHeight) {
 export function adaptTokenHolders(tokenHolders, tokenDetails) {
   const formattedData = tokenHolders.data.map(holder => ({
     address: holder.accountId,
-    amount: holder.amount / (10 ** tokenDetails.decimals),
-    percentage: (holder.amount / (10 ** (tokenDetails.decimals - 2))) / tokenDetails.totalSupply,
+    amount: (new BigNumber(holder.amount)).dividedBy(10 ** tokenDetails.decimals).toNumber(),
+    percentage: ((new BigNumber(holder.amount))
```

Comment:
Is there a need for so many brackets?

Reply:
1 set too much. I wonder why the linter didn't catch that. Thanks